lib/support_rota_client.rb

```diff
@@ -0,0 +1,22 @@
+class SupportRotaClient
+  def initialize
+    @api_base = if Rails.env.development? || Rails.env.test?
+                  "http://localhost:3000"
+                else
+                  "https://dxw-support-rota.herokuapp.com"
+                end
```

Comment:
Why do we flip based on `RAILS_ENV` rather than something more intentional? Do we need to flip between them at all? Can we pass the API url in as an environment variable?

Reply:
Oops! That was a quick temporary fix to have local data while the API was in review. 🙈 

We could revert to hardcoding the Heroku link, or use an env var. Probably better to env var it, to be more flexible if we ever move the support_rota app from Heroku or rename it.

Reply:
Sounds good :).

Reply:
Also supports working against a local or staging version if needed.